### PR TITLE
Update workflow to execute notebooks

### PR DESCRIPTION
**Pull request by maxrjones**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -17,65 +18,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.9', '3.10'] # run tests on multiple Python versions
+        python-version: ['3.10']
```
Comment (maxrjones) [inline .github/workflows/main.yaml]: @orianac can you update this to whichever Python version you're using?

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 carbonplan[data]
-zarr
-intake
-intake-xarray
+geopandas
+jupyterlab
+numpy
+matplotlib
+pandas
+xarray
```
Comment (maxrjones) [inline requirements.txt]: @orianac please pin these to the versions you're using

File: .github/workflows/main.yaml (hunk 1 above)
Comment (orianac) [reply .github/workflows/main.yaml]: great thanks

File: requirements.txt (hunk 2 above)
Comment (orianac) [reply requirements.txt]: will do

**Review by orianac — COMMENTED** [review]
thanks for the PR @maxrjones. i'll update as you flagged

**Review by orianac — COMMENTED** [review]
looks great! thanks @maxrjones

**Pull request merged**